app/lib/widgets/gallery/gallery.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:flex_ui/flex_ui.dart';
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:widgetbook_annotation/widgetbook_annotation.dart' as widgetbook;
+
+class FlexGallery extends StatefulWidget {
+  const FlexGallery({
+    super.key,
+    this.imageUrls = const [],
+    this.thumbnailUrls = const [],
+    this.borderRadius = 0,
+  });
+
+  final List<String> imageUrls;
+  final List<String> thumbnailUrls;
+  final double borderRadius;
+  // todo: expose many more properties for customization
```

Comment:
for future - Use flex image so we can leverage its placeholder option?

Reply:
Are you thinking `List<FlexImage> images` vs. sending in the URLs?

Reply:
If anything just an optional placeholder prop passed through to to the underlying FlexImages?
I honestly like the simplicity of this just taking in urls, and not being dependent on another Widget to use 

Could be just a ``MainImagePlaceholder``, and ``Thumbnailplaceholder`` - at this size/use I don't know if each thumbnail really needs a unique placeholder unless there's some big semantic benefit.